README.md

```diff
@@ -177,6 +177,11 @@ npm run dev -- -attach-validator
 ```
 - `-a` can also be used.
 
+Roosevelt makes use of an ENV variable to set the validator to an attached or detached state by default. 
+- Set an environment variable `ROOSEVELT_IS_DETACHED` to true for a detached by default validator.
```

Comment:
change this to `ROOSEVELT_VALIDATOR` with values "detached" or "attached"

note that when this env var is set, it overrides whatever setting is in package.json or params

Reply:
Should the env override the CLI as well?

Reply:
the hierarchy should be this:

- package.json
- params override package.json
- env vars override params
- CLI overrides env vars

might takes some sourceParams refactoring to make it behave this way iirc

Reply:
ENV need higher priority. If you want to make params have priority over that's fine but ENV have to be over package.json

Reply:
i think you misread that. it says env should have higher priority over params

as for CLI vs env, CLI should take higher priority because its purpose is to override anywhere else is set, package.json, app params, etc... that includes env

Reply:
think of it like this: CLI is used to force things. if the user sets something in their CLI command, they *really mean it*. so it should take priority over everything

Reply:
@Autre31415 do we need to refactor the sourceParams stuff to get it to override in the above hierarchy? if so, i'd like to do it as part of this PR

Reply:
@kethinov on my end it is currently working in the hierarchy you desire. the env variable will attach/detach as it's value states but will be overridden by CLI.

For example, having `ROOSEVELT_VALIDATOR` set to `detached`, then running the command `npm run dev -- -a` will force the `detached` state set by the ENV var to be overridden by the `-a` flag.